client/js/components/procedure/SegmentsList/CustomSearch.vue

```diff
@@ -61,14 +64,16 @@
           <dp-details
             v-for="explanation in explanations"
             :key="explanation.title"
-            :summary="explanation.title">
+            :summary="explanation.title"
+            :data-cy="'customSearch:' + explanation.title">
```

Comment:
- i think we can use es6 string literal instead of concatenation
- `customSearch` is too vague, i suggest `customSearchExplanation`
- title contains spaces, so it would be better to just add `(explanations, i)` in line 65 and append the index instead of the title here

Reply:
I think we don't need hooks here!
https://github.com/demos-europe/demosplan-core/pull/2587/commits/0ac8f7ce8160a1a27e21e25300e770c0f569e60e

Reply:
because here no actions, only data and infos!

Reply:
I think we don't need that in E-2-E!

Reply:
ok, that's right, i understand. maybe we want a jest test of DpDetails in demosplan-ui, but there is no need for an e2e test here.